test/util/api_util.js

```diff
@@ -10,12 +10,15 @@ function getPublicUserFromUser(user) {
   };
 }
 
-async function createGameAndAssert(t, api, userCred, user) {
+async function createGameAndAssert(t, api, userCred, user, gameOptionalInfo = {}) {
   // TODO: we should switch from using tictactoe to runner/test_app backend where we
   // have more control over the backend behavior. We can force trigger errors, or other behaviors
+  const name = gameOptionalInfo.name || `integration-tests-${uuidv4()}`;
+  const description = gameOptionalInfo.description || 'test description';
```

Comment:
```suggestion
  const description = gameOptionalInfo.description ?? 'test description';
```
I think the nullish coalescing operator is the correct one here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator